lib/circuitbox.rb

```diff
@@ -7,6 +7,9 @@
 require 'circuitbox/version'
 require 'circuitbox/circuit_breaker'
 require 'circuitbox/notifier'
+require 'circuitbox/timer/null_timer'
+require 'circuitbox/timer/monotonic_timer'
+require 'circuitbox/timer/simple_timer'
```

Comment:
Maybe we should only load the default one and then let users require the ones they need?